website/content/docs/enterprise/admin-partitions.mdx

```diff
@@ -95,7 +95,9 @@ One of the primary use cases for admin partitions is for enabling a service mesh
 - A Consul Enterprise license must be installed on each Kubernetes cluster.
 - The helm chart for consul-k8s v0.39.0 or greater.
 - Consul 1.11.1-ent or greater.
-- All Consul clients must be able to communicate with the Consul servers in the `default` partition, and all servers must be able to communicate with the clients.
+- All Consul clients must be able to communicate with the Consul servers in the `default` partition, and all servers must be able to communicate with the clients. 
+  - A designated Kubernetes `LoadBalancer` service is exposed on the Consul server cluster to allow communication for RPC (port 8300), Gossip (port 8301), and HTTPS (port 443) API requests to the both the Consul servers and the `default` partition. 
+  - Mesh Gateways are deployed as a Kubernetes `LoadBalancer` service (on port 443) across all Kubernetes clusters to allow for cross partition networking as described in [Cross-Partition Networking](#cross-partition-networking). 
```

Comment:
```suggestion
- A designated Kubernetes `LoadBalancer` service must be exposed on the Consul server cluster. This enable the following communication channels to the Consul servers and the `default` partition:
  - RPC on port 8300 
  - Gossip on port 8301
  - HTTPS API requests on port 443 API requests 
- Mesh gateways must be deployed as a Kubernetes `LoadBalancer` service on port 443 across all Kubernetes clusters.
- Cross-partition networking must be implemented as described in [Cross-Partition Networking](#cross-partition-networking). 
```
Trying to make this more bullet-y list-y so it's easier to consume while still providing all of the information.  
  
This seems like a general requirement: 

- All Consul clients must be able to communicate with the Consul servers in the `default` partition, and all servers must be able to communicate with the clients. 

Maybe move it up the top? Or create a Networking sub requirement and put it there?

Reply:
We can move up higher as a general networking requirement under its own section.